docs/operational_guide/topology.md

```diff
@@ -0,0 +1,173 @@
+# Topology
+
+## Overview
+
+M3DB stores its topology (mapping of which hosts are responsible for which shards) in EtcD. There are three possible states that each host/shard pair can be in:
```

Comment:
Nit: `s/EtcD/etcd`. Maybe link to their docs as well? https://coreos.com/etcd/

Reply:
I wonder if we should use the word "placement" more often here instead of, or in addition to, topology. For example in m3cluster everything is a placement, and we're storing placements in etcd.